### DISCUSSION TITLE
Using Crazyradio with other receiver protocols.

### DISCUSSION THREAD

**Discussion by dmj17b** [bitcraze/discussions#109] [Q&A]
Hello,

I am working on a nano fixed wing aircraft that will rely on a motion capture system to do position feedback control. It is extremely easy to find sub 1g receivers, but they only work with DSM2/DSMX frequency hopping protocol. What I would like to know if it is possible to re-program the crazyradio to connect to such a frequency hopping system. Otherwise, are there any sub 1g receivers that I can use with the crazyradio?

Thanks

-DJ

**Comment by tobbeanton** [comment]
The Crazyradio is based on the Nordic Semi nRF MCUs. I doubt they could be made compatible with the DSM2/DSMX protocol that are based on the Cypress radio MCUs.

You could use a NRF24 SPI dongle (probably not sub 1g) but I doubt that is what you want since you would have to implement the CRTP stack on your controller... 

I'm not really sure what you want to do as the DSM2/DSMX protocols only send channel data, not possible to send other data such as position?

**Reply by dmj17b** [reply]
Basically, my professor was super happy using your small crazyflie drones in our motion capture volume, and now he wants to work on micro fixed wing aircraft. My plan was to get all of the position data from the motion capture system, process it from a ground station computer, and then send strictly servo inputs to the aircraft rather than trajectories. This idea is just supposed to be a proof of concept and will not have any onboard logic like the CrazyFlie drones themselves. I just can't seem to find any 2.4ghz transmission system that can connect to a ground computer like the CrazyRadio.

**Reply by tobbeanton** [reply]
Possibly some of the R/C radios has a computer interface you can control them with. Maybe checkout OpenTX or EdgeTX open source transmitter projects. Found [this](https://drones.stackexchange.com/questions/1321/control-opentx-transmitter-from-pc) when doing a quick search.
